ChatSchema.md

```diff
@@ -0,0 +1,105 @@
+# Chat Server-side Schema
+
+## Entity Design
+
+### User
+
+User entity includes:
+
+- user_id: identify a user. (Pk)
+- name: shown name.
+- password: login password (optional).
+- create_time: create time.
+- update_time: update time.
+- visual_id: linked visual charactor.
+- data: map(str->str) containing additional information like last login.
```

Comment:
I thought we can always add new fields to the entities? Having a universal map field seems a little bit un-safe. For example, if we would like to track last login time, we can add a last_login_time of type timestamp.